emberjs/README.md

```diff
@@ -0,0 +1,52 @@
+# emberjs
```

Comment:
Maybe a little description on where to find the Wootric files and what's the project about? I think that'll help as guidance for whoever wants to use this as reference